src/Blocks/NavigationTab.purs

```diff
@@ -21,7 +21,7 @@ type IsActive = Boolean
 tabClasses :: Array HH.ClassName
 tabClasses = HH.ClassName <$>
   [ "leading-normal"
-  , "mr-6"
+  , "mx-3"
   , "mt-4"
```

Comment:
i think this will cause the left most tab to be offset from the edge of the form.